utilities/merge_operators/put.cc

```diff
@@ -8,10 +8,12 @@
 #include "rocksdb/merge_operator.h"
 #include "utilities/merge_operators.h"
 
-using namespace ROCKSDB_NAMESPACE;
-
 namespace { // anonymous namespace
 
+using ROCKSDB_NAMESPACE::Logger;
+using ROCKSDB_NAMESPACE::MergeOperator;
+using ROCKSDB_NAMESPACE::Slice;
+
```

Comment:
Why not put everything inside a namespace ROCKSDB_NAMESPACE { like the rest of the code?

Reply:
This PR aims to be a simple, mechanical, straightforward change, trying to adhere to previous code behavior as much as possible except for getting rid of `using namespace`. It will be easier for both author and reviewers to reason about possible consequences of the change, especially if the changes span across 20+ files.
If you believe the proposal is an improvement, it can be done in a separate PR.